app/src/lib/stores/git-store.ts

```diff
@@ -1647,18 +1648,15 @@ export class GitStore extends BaseStore {
 
   public async pruneForkedRemotes(openPRs: ReadonlyArray<PullRequest>) {
     const remotes = await getRemotes(this.repository)
-    const prRemotes = new Set<string>()
 
-    for (const pr of openPRs) {
-      if (pr.head.gitHubRepository.cloneURL !== null) {
-        prRemotes.add(pr.head.gitHubRepository.cloneURL)
-      }
-    }
+    const remotesToPrune = findForkedRemotesToPrune(
+      remotes,
+      openPRs,
+      this.allBranches
+    )
```

Comment:
If we're being a little sneaky and grabbing a reference to allBranches here we can tighten this up a bit.

```suggestion
    const branches = this.allBranches
    const remotesToPrune = findForkedRemotesToPrune(remotes, openPRs, branches)
```